public/docs/ts/latest/guide/router.jade

```diff
@@ -770,6 +777,42 @@ a#get-route-parameter
   The router extracts the route parameter (`id:15`) from the URL and supplies it to
   the `HeroDetailComponent` via the **ActivatedRoute** service.
 
+<a id="activated-route"></a>
+h3#activated-route ActivatedRoute: the one-stop-shop for route information
+:marked
+  Each route contains information about its path, data parameters, URL segment and much more.
+  All of this information is available in an injected service provided by the router called the [ActivatedRoute](../api/router/index/ActivatedRoute-interface.html).
+
+  The `ActivatedRoute` contains all the information you need from the current route component as well as ways to get information
+  about other activated routes in the `RouterState`.
+
+.l-sub-section
+  :marked
+    **`url`**: An `Observable` of the route path(s). The value is provided as an array of strings for each part of the route path.
+
+    **`data`**: An `Observable` that contains the `data` object provided for the route. Also contained any resolved values from the [resolve guard](#resolve-guard).
+
+    **`params`**: An `Observable` that contains the required and [optional parameters](#optional-route-parameters) specific to the route.
+
+    **`queryParams`**: An `Observable` that contains the [query parameters](#query-parameters) available to all routes.
+
+    **`fragment`**:  An `Observable` of the URL [fragment](#fragment) available to all routes.
+
+    **`outlet`**: The name of the `RouterOutlet` used to render the route. For an unnamed outlet, the outlet name is **primary**.
+
+    **`routeConfig`**: The route configuration used for the route that contains the origin path.
+
+    **`parent`**: an `ActivatedRoute` that contains the information from the parent route when using [child routes](#child-routing-component).
+
+    **`firstChild`**: contains the first `ActivatedRoute` in the list of child routes.
+
+    **`children`**: contains all the [child routes](#child-routing-component) activated under the current route.
+
+:marked
+  We import the `Router`, `ActivatedRoute`, and `Params` tokens from the router package.
+
++makeExcerpt('app/heroes/hero-detail.component.1.ts (activated route)', 'imports')
```

Comment:
I don't think so. The service does a strict equality check so if a letter is passed in, it won't find a match